packages/cli/src/checks/utils.ts

```diff
@@ -52,7 +50,7 @@ type AllCheck<ErrorType> = {
   validate: (
     workspace: Package,
     allWorkspaces: Map<string, Package>,
```

Comment:
this wasn't introduced by this PR so we can clean this up later - but I think we should call those either workspaces or packages, right now the codebase is using two different names for the same thing